ecs-init/volumes/ecs_volume_plugin.go

```diff
@@ -129,11 +129,13 @@ func (a *AmazonECSVolumePlugin) Create(r *volume.CreateRequest) error {
 	}
 
 	// get driver type from options to get the corresponding volume driver
-	var driverType string
+	var driverType, target string
```

Comment:
Does this mean customer can specify a target path to specify where they want to mount the volume? Or agent is the one who decides the target?

Reply:
for EFS, agent doesnt pass in target and plugin creates the mount path. 
This is just added for mostly testing purposes

Reply:
What kind of testing?

Reply:
if we don't want to mess with the original ECS volumes dir. example: the mount path created by plugin is /var/lib/ecs/volumes/<volume-name> . I can override it using the target field and have /foo as mount